### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

File: doc/src/methodology.ltx [hunk 1]
```diff
@@ -84,22 +84,25 @@ information was removed, with the caveat that a professor manually review
 anonymized code to ensure that no obvious identifying information remained.
 Professor Lauer graciously offered to perform this final vetting.
 
-% TODO add bash script as listing to report
+\begin{verbatim}
+# TODO; the anonymization script, in full, will go here
+\end{verbatim}
```
Comment (mheon) [inline doc/src/methodology.ltx]: You don't want to do this. It's hundreds of lines. http://sed.sourceforge.net/grabbag/scripts/remccoms3.sed is the source for just the comment stripping; the directory handling logic is another hundred or so lines of Bash.

File: doc/src/methodology.ltx [hunk 2]
```diff
@@ -84,22 +84,25 @@ information was removed, with the caveat that a professor manually review
...
 A script was constructed in Bash to accomplish this goal in a largely-automated
 fashion to make it feasible to obtain large bodies of test code. Given the
 nature of the script, its use is limited to submissions written in languages
 that use comments delineated with the ``\texttt{//}'' and ``\texttt{/* */}''
-symbols.
-
-% TODO add citation for remcomms sed script
+symbols. The anonymization script used signiicant parts of the
+\textit{remcomms} Sed script for removing C comments, developed by Stewart
+Ravenhall.
```
Comment (mheon) [inline doc/src/methodology.ltx]: No, we literally just call the original script without modification, so feel free to say that.

And Stewart Ravenhall is not the author of the script. The current developer is Brian Hiles, building off work by Paolo Bonzini.

Comment (dmurvihill) [inline doc/src/methodology.ltx]: Okay, I will change it to say that we used the original script; it is [this one](https://gist.github.com/mystix/426760), right?

Comment (mheon) [inline doc/src/methodology.ltx]: No, http://sed.sourceforge.net/grabbag/scripts/remccoms3.sed is the script we used.

**Comment by dmurvihill** [comment]
okay, should be ready to go.

File: doc/src/methodology.ltx (hunk 1 above)
Comment (dmurvihill) [inline doc/src/methodology.ltx]: Sadly, Lauer wants it.

**Pull request merged**